src/js/weather.js

```diff
@@ -0,0 +1,103 @@
+import { renderSearch } from './render';
+
+const weatherApp = (params) => {
+  const { city } = params;
+  const url = 'http://api.openweathermap.org/data/2.5/forecast?q=';
+  const id = '&APPID=092878e572bdc25b4b5bdea6cbd439db';
+  const cel = '&units=metric';
+  const forecastPromiseCel = fetch(`${url}${city}${cel}${id}`, { mode: 'cors' });
+
+  const setCache = (response) => {
```

Comment:
Very nice setup for the cache, did you test it thoroughly to check it is working as expected?

Reply:
Yes, I check every step outputting the information in the console. It works to check if there's a city, then when was it stored and then removes it or loads the info.